### PR TITLE
MAKE MAPP GO ZOOM ZOOM

### PR DESCRIPTION
**Pull request by bluemode**
## Description
MAPP SHOULD GO ZOOM ZOOM! I REMOVE STUFF THAT MAKE MAP GO _slow slow_

## Motivation and Context
MAP DOES NOT EVEN GO ONE ZOOM

## How Has This Been Tested?
NO MORE WEIGHT, MAP CAN ZOOM ZOOM

## Types of changes
smol change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by notafakeaccount69** [comment]
+1

**Comment by FrostTheFox** [comment]
Shit PR, travis failing, code quality goes down. Please fix, rebase, and you'll have my +1.

**Comment by elincognito** [comment]
Testing as soon as I can, should I `run rm - rf` first?

**Comment by elincognito** [comment]
@bluemode Travis don't like no one,

File: docs/extras/captchas.md [hunk 1]
```diff
@@ -1,92 +0,0 @@
...
-**Remember**: Status name (`-sn` / `--status-name`) is required for RocketMap to store account statistics in the database, otherwise the captcha page will keep displaying zeros.
-
-### Extra Parameter: `--manual-captcha-refresh`
-Simply put, this is an easy way of controlling how often you want the captcha solving page to be refreshed.
-
-Captcha tokens (the result of solving a captcha) are only valid for a limited amount of time. This validity period starts a little bit after the bookmarklet loads, when recaptcha code is also loaded. This means that if the captcha is solved after a minute or two of browser idle time, the resulting token will not verify challenge and account will remain on hold.
-
-- Manual captcha refresh: `-mcr 30` / `--manual-captcha-refresh 30`
-
-The default value is `30` seconds because we had good results with it during our tests.
```
Comment (sebastienvercammen) [inline docs/extras/captchas.md]: If removed, what's new default??!!? Needs clarify, otherwise good PR 👍

**Comment by sebastienvercammen** [comment]
Pls rebase

File: docs/extras/captchas.md (hunk 1 above)
Comment (bluemode) [inline docs/extras/captchas.md]: With ZOOM ZOOM, we don't get captchas any more. No need to slow map down by worrying about refreshing captcha pages!!

**Comment by elincognito** [comment]
This pr is amazing got almost 0% Cpu usage on it

**Comment by neskk** [comment]
Some people have way too much free time 💃

**Comment by FrostTheFox** [comment]
(for realzies though zoom zoom must go :( )

**Comment by sebastienvercammen** [comment]
Need more ZOOM ZOOM in our lives

**Pull request closed without merging**